.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
```

Comment:
Let's add explicit test commands to this Travis config file, e.g. https://github.com/goodeggs/fake-eggs/blob/1ccc0ac5760c63074a066d602758837883b993c0/.travis.yml#L20-L39.

Although since it's so basic already, consider just moving to Buildkite (although ask Bob... I think we haven't moved any _public_ repos yet...).

Reply:
Makes sense, Ill propose to Bob to be the next one to migrate to buildkite